src/test/scala/play/extras/geojson/SphericalMercatorSpec.scala

```diff
@@ -0,0 +1,37 @@
+package play.extras.geojson
+
+import scala.collection.immutable.Seq
+import org.specs2.mutable.Specification
+import play.api.libs.json._
+
+class DistancesTest extends Specification {
+
+  "SphericalMercator.fromEPSG4326" should {
+
+    "return LatLng(1290504.0696666553, 6132301.564388968)" in {
+      val epsg3857 = SphericalMercator.fromEPSG4326(LatLng(48.1527311, 11.5927953))
+      epsg3857 mustEqual SphericalMercator(1290504.0696666553, 6132301.564388968)
```

Comment:
Technically, these tests may pass or fail depending on what CPU they run on.  In order for them to pass on all architectures, you'd need to annotate every double `val` with `@fpstrict`, which instructs the JVM and in turn the CPU to use strict floating point calculations, which guarantees exactly the same results on all architectures.

A better thing to do here would be to do matching using specs's `beCloseTo` matcher, matching to 10 significant figures or something like that.